### ISSUE TITLE
Doxygen: Find a workaround for packages with EXTRACT_ALL=ON

### ISSUE THREAD

**Issue by sloriot** [CGAL/cgal#1799]
in the official doxygen if `EXTRACT_ALL=ON` everything has a long description. We cannot make the patch do upstream. A solution is to set this flag OFF in all packages relying on it and add some documentation to members not documented that won't go into the doc with the flag change. This should be done with https://github.com/CGAL/doxygen/tree/release_1_8_13_patched as previous version were buggy.

See https://github.com/doxygen/doxygen/pull/452

**Comment by lrineau** [comment]
In my opinion we should first design tools allowing to compare two produced documentations.

**Comment by sloriot** [comment]
Not necessarily, as I said the comparison for a fixed version of doxygen can be done if `HTML_TIMESTAMP=OFF` with diff.

**Comment by lrineau** [comment]
If I understand well:

Maxime can worked with our fork of Doxygen, configure with `HTML_TIMESTAMP=OFF`, and then compare two builds: one with `EXTRACT_ALL=ON` and one without it.

Am I right?

**Comment by sloriot** [comment]
Exactly.

**Comment by sloriot** [comment]
However, 1.8.13 fork should be used as I notice some bugs in the other version when putting `EXTRACT_ALL=OFF` for some enum types.

**Comment by sloriot** [comment]
IMO, the priority of this issue is low as this regular version of doxygen can be used, it just generate an extra useless long description. Moreover Dimitri mentioned that he is working on a way to select what to export, so there might be another solution coming soon.

**Comment by maxGimeno** [comment]
@sloriot Is there any news on this ?

**Comment by sloriot** [comment]
No update on doxygen side. Maybe we should go on with always setting `EXTRACT_ALL` to `OFF`.

**Comment by albert-github** [comment]
Recently a number of changes have been made in the determination whether or not a detailed section should be present.
There has been a change in the function `bool MemberDefImpl::isDetailedSectionLinkable() const`, especially also in the handling in respect for `EXTRACT_ALL` see https://github.com/doxygen/doxygen/pull/8601
In the code I have in the doxygen repository the only  change is what I see in https://github.com/doxygen/doxygen/pull/452.

It would be good to see:
- what the differences are with and without the patch for the current master.
